mesos-swarm-marathon/configure-mesos-cluster.sh

```diff
@@ -11,6 +11,7 @@
 ###########################################################
 
 echo "starting mesos cluster configuration"
```

Comment:
On further discussion, will continue to hold off a bit longer on fast fail (but not ruling out long term) including "set -e" and "set -o pipefail".  Will choose "set -x" as a benefit to see exactly what is being called.